src/diagnose.ts

```diff
@@ -16,16 +16,16 @@ const placeholderNameRegex = /^[a-zA-Z][a-zA-Z_$0-9]*$/; //Must be able to trans
 const keyNameRegex = /^[a-zA-Z][a-zA-Z_0-9]*$/; //Must be able to translate to a (non-private) Dart method
 
 export enum DiagnosticCode {
-	mismatchedBrackets,
-	metadataForMissingKey,
-	invalidKey,
-	missingMetadataForKey,
-	invalidPlaceholder,
-	missingOtherInICU,
-	unknownICUMessageType,
-	placeholderWithoutMetadata,
-	missingPlaceholderWithMetadata,
-	missingMessagesFromTemplate,
+	mismatchedBrackets = "mismatched_brackets",
```

Comment:
Nit: Any reason to introduce the underscore separated strings, and not simply use the names themselves?

Reply:
I saw that the dart plugin uses snake case for diagnostic codes, so I used that for consistency